app/models/katello/host/subscription_facet.rb

```diff
@@ -34,6 +35,19 @@ def import_database_attributes(consumer_params)
         end
       end
 
+      def update_consumer_type(consumer_params)
+        if consumer_params.try(:[], 'type').try(:[], 'label') == 'hypervisor'
+          self.consumer_type = 'hypervisor'
+        elsif consumer_params.try(:[], 'facts').try(:[], 'virt.is_guest') == "true"
```

Comment:
I'd like to be able to search for "VMs with no hypervisor" and "VMs with a hypervisor". If you want to call all VMs guests, I guess that's alright but we use the terms hypervisor and guest in addition to physical and virtual. To me guest and virtual are different: A guest is a virtual with a known hypervisor.